src/solve2!.jl

```diff
@@ -0,0 +1,80 @@
+function create_context(::Problem{T}, optimizer) where {T}
+    model = MOIB.full_bridge_optimizer(MOIU.CachingOptimizer(MOIU.UniversalFallback(MOIU.Model{T}()),
+                                                             optimizer), T)
+
+    return (var_id_to_moi_indices=OrderedDict{UInt64,Vector{MOI.VariableIndex}}(),
```

Comment:
this could be a `struct` instead of a `NamedTuple`, but for prototyping I think it's easier as a NamedTuple